config/sync/core.extension.yml

```diff
@@ -106,6 +106,9 @@ module:
   uhsg_some_links: 0
   uhsg_themes: 0
   uhsg_user_sync: 0
+  update: 0
+  upgrade_rector: 0
+  upgrade_status: 0
```

Comment:
You are right of course. The modules should not cause additional load on normal bootstraps. I thought about running once them in production also because there seems to be some differences depending on env, PHP version etc, but the modules can be enabled manually them. 

The error you receive is a bit surprising because I have the same file in the same location and dont get the error. Might be about autoloading order, caches or something. But it should be avoidable with this addition to exclude section of rector.yml:

```
  exclude_paths:
    - '/app/build/vendor/symfony/dependency-injection/Tests/Fixtures/ParentNotExists.php'
```

I'll fix this and a few other things in a commit.